packages/docs/celo-codebase/protocol/identity/phone-number-privacy.md

```diff
@@ -1,46 +1,50 @@
 # Phone Number Privacy
 
-An important part of the Celo protocol is the ability to link a phone number to a Celo native address. This improves the user experience for sending/receiving payments to a phone's contacts without having to deal with blockchain native addresses. However, it's also important that the phone number mappings remain private, preventing the mass harvesting of mapped phone numbers. This guide provides an overview of the Celo phone number privacy system.
+Celo's [identity protocol](README.md) allows users to associate their phone number with one or more addresses on the Celo blockchain. This allows users to find eachother on the Celo network via their phone number, rather than cumbersome hexadecimal addresses. The Phone Number Privacy (PNP) protocol helps preserve the privacy of users blockchain addresses by preventing the mass harvesting of the phone numbers of Celo users.
 
 ## Understanding the problem
 
-When you send payment to someone in your phone's address book, the mobile client needs to look up the phone number in the on-chain registry to find the mapping to a Celo native address, which can be used to receive the actual payment. If this mapping were in plain text on the Celo blockchain then anyone would be able to trivially see account balances for all numbers. If instead, the hash of each phone number were used as the on chain identifier, it would still be possible to brute force the mapping via a [rainbow table attack](https://en.wikipedia.org/wiki/Rainbow_table). 
+When a users sends a payment to someone in their phone's address book, the mobile client must look up the identifier for that phone number on-chain to find the corresponding Celo blockchain address, which is needed in order to complete the payment transaction. If the identifier was simply the recipients phone number, then anyone would be able to associate phone numbers with blockhain accounts and balances. If instead, the identifier was the hash of the recipient's phone number, attackers would still be able to associate phone numbers with blockchain accounts and balances via a [rainbow table attack](https://en.wikipedia.org/wiki/Rainbow_table).
```

Comment:
1st sentence: `users -> user`
2nd sentence: `recipients phone number -> recipient's phone number`
2nd sentence: `blockhain -> blockchain`

**Non-blocking Suggestions**

1. Split first sentence for readability:
`... to find the corresponding Celo blockchain address. This address is needed in order to complete the payment transaction.`

2. Resolve singular/plural mismatch in the second sentence:
```
singular: If the identifier was simply the recipient's phone number, anyone could associate it with its blockchain account and balance.

plural: If the identifiers were simply the recipients' phone numbers, anyone could associate them with their blockchain accounts and balances.
```

3. Resolve singular/plural mismatch in the third sentence:
```
singular: 
If instead, the identifier was the hash of the recipient's phone number, attackers would still be able to associate that phone number with its blockchain account and balance via a [rainbow table attack](https://en.wikipedia.org/wiki/Rainbow_table).

plural:
If instead, the identifiers were the hashes of the recipients' phone numbers, attackers would still be able to associate those phone numbers with their blockchain accounts and balances via a [rainbow table attack](https://en.wikipedia.org/wiki/Rainbow_table).
```